components/create/Skills.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import Chip from '@mui/material/Chip';
+import Stack from '@mui/material/Stack';
+import CreatePortfolioContext from 'components/create/CreatePorfolioContext';
+import { StyledTextField } from 'components/create/Styled';
+import { useState, useContext } from 'react';
+const Skills = () => {
+  const { skills, setSkills } = useContext(CreatePortfolioContext);
+  const [techSkill, setTechSkill] = useState('');
+  const [softSkill, setSoftSkill] = useState('');
+
+  const handleClick = () => {
+    console.info('You clicked the Chip.');
+  };
+  const handleDelete = () => {
+    console.info('You clicked the delete icon.');
+  };
+  return (
+    <div>
+      <StyledTextField
+        label={'Tech Skills'}
+        value={techSkill}
+        onChange={(event) => {
+          setTechSkill(event.target.value);
+        }}
+        onKeyPress={(e: any) => {
+          if (e.key === 'Enter') {
+            skills.tech?.push(e.target.value);
+            setSkills({ ...skills });
```

Comment:
You shouldn't be modifying a state variable like `skills` directly, this can cause bugs. Instead, use `setSkills` like:

```
setSkills({
  ...skills,
  ...tech: [...skills.tech, e.target.value]
})
```

Reply:
fixed